### PR TITLE
Guides improvements

### PR DESCRIPTION
**Pull request by Turolatias**
I added mentions to the message content intent, which is necessary to make some projects in the guides work

### CODE REVIEW

File: src/content/docs/guides/attachments.mdx [hunk 1]
```diff
@@ -86,6 +86,12 @@ luau supports sending these attachments through the `AttachmentBuilder` object.
    ```
 </details>
 
+:::caution
+   Remember to enable the message content intent on the [Discord Developer Portal](https://discord.com/developers) when the `DiscordClient.eventManager.onMessage` event is used. 
+   
+   If the intent is not enabled, the bot won't have access to the messagge content!
```
Comment (CompeyDev) [inline src/content/docs/guides/attachments.mdx]:
```suggestion
   If the intent is not enabled, the bot won't have access to the message content!
```

Minor typo.

**Review by CompeyDev — CHANGES_REQUESTED** [review]
(*something I noticed here was the use of `addEmbed` as opposed to `setEmbed`, is this how it's supposed to be used (cc @4x8Matrix)?*)

Everything else looks fine other than the small spelling mistake, can merge once fixed.

**Comment by 4x8Matrix** [comment]
Yes, we are supposed to use the addEmbed method, not setEmbed @CompeyDev ! :sweat_smile:

**Review by CompeyDev — APPROVED** [review]
LGTM!

**Pull request merged**